src/bridge/views/SentTransaction.js

```diff
@@ -148,7 +146,6 @@ function SentTransaction(props) {
         <Button
           prop-type={'link'}
           onClick={() => {
-            setPointCursor(pointCursor);
```

Comment:
this line didn't seem like it was doing anything, since `setPointCursor(pointCursor)` would just produce the same state. @Fang- any insight here?

Reply:
That does indeed look like a no-op. And one that's been around since forever at that! Seems fine to take it out if it doesn't break anything.